### DISCUSSION TITLE
bookmarklet executer

### DISCUSSION THREAD

**Discussion by Turbored80** [3kh0/ext-remover#1172] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there a video for the bookmarklet executer because every time I try I end up with it not working. 
![Screenshot 2024-09-18 3 56 16 PM](https://github.com/user-attachments/assets/f587ce8f-fb72-4d90-87b1-39119d7d4636) I can't get past this screen. I have light speed as my blocker.

**Comment by AshtonDavies** [comment, marked as answer]
What could you possibly need it for?

**Comment by HelloIloveGimkit2** [comment]
> Is there a video for the bookmarklet executer because every time I try I end up with it not working. ![Screenshot 2024-09-18 3 56 16 PM](https://private-user-images.githubusercontent.com/179605271/368767463-f587ce8f-fb72-4d90-87b1-39119d7d4636.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY3NDc5MTEsIm5iZiI6MTcyNjc0NzYxMSwicGF0aCI6Ii8xNzk2MDUyNzEvMzY4NzY3NDYzLWY1ODdjZThmLWZiNzItNGQ5MC04N2IxLTM5MTE5ZDdkNDYzNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxOVQxMjA2NTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hNGJiYzkzNTAwYWY4NmQ0MzUyZTM5NWNhZDU0MTFlMTEwMzk5NzM2NmY4NjM3YTU1NjNkNmRhZTJkMTkxYmM5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.k7VyotkeOyHWuFGo00ogMuQuW7gPqR-K-bWYMdc-NWg) I can't get past this screen. I have light speed as my blocker.

at that screen click the page so that it's focused then type "thisisunsafe" no spaces

**Reply by AshtonDavies** [reply]
The DNS is useless. You all should know that if you are reading the discussion for it.
